modules/openfast-library/src/FAST_Library.f90

```diff
@@ -497,6 +497,10 @@ subroutine FAST_OpFM_Init(iTurb, TMax, InputFileName_c, TurbID, NumSC2Ctrl, NumC
    ExternInitData%NumActForcePtsTower = NumActForcePtsTower
 
    CALL FAST_InitializeAll_T( t_initial, 1_IntKi, Turbine(iTurb), ErrStat, ErrMsg, InputFileName, ExternInitData )
+
+   IF ( ErrStat >= AbortErrLev ) THEN
+      CALL ProgAbort( "Error in FAST_OpFM_Init:FAST_InitializeAll_T" // TRIM(ErrMsg) )
+   END IF
    
```

Comment:
Wouldn't it be better to return to the calling routine after line 511 instead of abruptly ending the program here? That way memory can be deallocated and files closed.

Reply:
Yes.  Good idea.  @ewquon updated this.